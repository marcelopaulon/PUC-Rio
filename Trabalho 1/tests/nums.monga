/* int */

0x0
0X1
0x02
0x84848

1
9
98
098

/* double */

0.4
0.5
1.0
1.5
1.7
900.0
900.07
900.141414151515151515151515
900.2e23
2.0e2

0x1.0p+3
0x1.0p3
0x1.0p-3
0x0.0
0x6.7
0x6.7P+9
0x6.7P9
0x6.7P-9

ERROS

0x.0p+3
0x0.0+3
0x0.
0x1.0-3
0x.0P+3
