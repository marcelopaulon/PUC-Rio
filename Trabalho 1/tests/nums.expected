START
272(3)0
272(4)1
272(5)2
272(6)542792
272(8)1
272(9)9
272(10)98
272(11)98
272(15)97
272(16)67
272(17)92
272(18)0
272(19)0
272(20)254
272(21)2
272(22)41
271(26)0x1.999999999999ap-2 / 0.400000
271(27)0x1p-1 / 0.500000
271(28)0x1p+0 / 1.000000
271(29)0x1.8p+0 / 1.500000
271(30)0x1.b333333333333p+0 / 1.700000
271(31)0x1.c2p+9 / 900.000000
271(32)0x1.c208f5c28f5c3p+9 / 900.070000
271(33)0x1.c21219dbe1f98p+9 / 900.141414
271(35)0x1p+3 / 8.000000
271(36)0x1p+3 / 8.000000
271(37)0x1p-3 / 0.125000
271(38)0x0p+0 / 0.000000
271(39)0x1.9cp+2 / 6.437500
271(40)0x1.9cp+11 / 3296.000000
271(41)0x1.9cp+11 / 3296.000000
271(42)0x1.9cp-7 / 0.012573
269(44)ERROS
272(46)0
269(46)x
46(46)
272(46)0
269(46)p
43(46)
272(46)3
271(47)0x0p+0 / 0.000000
43(47)
272(47)3
272(48)0
46(48)
271(49)0x1p+0 / 1.000000
45(49)
272(49)3
272(50)0
269(50)x
46(50)
272(50)0
269(50)P
43(50)
272(50)3
END
